app/pods/publications/publication/proofs/documents/template.hbs

```diff
@@ -1,4 +1,10 @@
 <div class="auk-u-m-4">
+  <Publications::Publication::Proofs::ProofInfoPanel
+    @publicationSubcase={{this.publicationSubcase}}
+    @publicationFlow={{this.publicationFlow}}
```

Comment:
De `@publicationFlow` wordt hier enkel meegegeven als argument om er in de component dan de status van op te halen. Aangezien `@publicationSubcase` apart wordt meegegeven, zou ik dat dan ook verwachten van de status. Of misschien zelfs nog een stap verder en enkel `@isPublished={{this.publicationFlow.status.isPublished}}` meegeven, want dat is het enige wat ervan gebruikt wordt in `<Publications::Publication::Proofs::ProofInfoPanel>`

Reply:
`this.publicationFlow.status.isFinal` is waarschijnlijk beter als check zodat de overdue warning ook niet getoond wordt bij teruggetrokken publicaties.